src/libraries/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.TryGet.cs

```diff
@@ -68,11 +68,13 @@ namespace System.Text.Json
         /// <exception cref="ArgumentException">The destination buffer is too small to hold the unescaped value.</exception>
         public readonly int CopyString(Span<byte> utf8Destination)
         {
-            if (_tokenType is not (JsonTokenType.String or JsonTokenType.PropertyName))
+            if (_tokenType is not (JsonTokenType.String or JsonTokenType.PropertyName or JsonTokenType.Number))
```

Comment:
Is this actually what we want? It doesn't seem particularly intuitive that GetString works for numbers...

If this is what we want we should update the xml doc

Reply:
I see your point, in hindisight this should probably have been named `CopyValue` to avoid confusion. My rationalization for this change is that 1) it unblocks a useful scenario and 2) JSON numbers are technically string representations without quote marks.

> If this is what we want we should update the xml doc

Yes, absolutely 👍

Reply:
@krwq I got the same concern while working on it. If we want to avoid making a change that we then will regret, we could just make an `internal CopyValue(Span<byte>)` helper. 

I don't have a strong opinion, though.

Reply:
I'm ok with using an internal helper for now.